src/sentry/api/validators/sentry_apps/schema.py

```diff
@@ -177,9 +178,58 @@
     "required": ["elements"],
 }
 
+element_types = ["issue-link", "alert-rule-action", "issue-media", "stacktrace-link"]
+
+
```

Comment:
Note that `validate_component` already existed. I just moved it from a test util file to here so we can use it outside of tests.